common/helper.py

```diff
@@ -45,6 +47,27 @@ class UnsupportedArchiveError(Exception):
     pass
 
 
+class ErrorCode(Enum):
+    """
+    Container for custom error codes
+    """
+
+    CRITICAL = 1
+
+
+class Stage(Enum):
```

Comment:
I think it is used by build scripts only

Reply:
But we can use it not only in build scripts.